src/main/fc/stats.c

```diff
@@ -0,0 +1,75 @@
+
+#include "platform.h"
+
+#ifdef USE_STATS
```

Comment:
I think the name of this is easy to be confused with the post-flight stats in OSD. Maybe call this `USE_PERSISTED_STATS`.

Reply:
ok